streamwebs_frontend/streamwebs/views.py

```diff
@@ -24,11 +24,203 @@
+            'somewhat_sensitive_total': 'somewhat sensitive total',
+            'aquatic_worm': 'aquatic worm', 'mosquito_larva': 'mosquito larva',
+            'tolerant_total': 'tolerant total',
+            'wq_rating': 'water quality rating'
+        }
+    )
+
+
+# Come back to this to fix schools and foreign key calls
+def export_ript(request, site_slug):
+    site = Site.objects.get(site_slug=site_slug)
+    rip_transect = RiparianTransect.objects.filter(site_id=site.id)
+
+    sheets = []
+    for each in rip_transect:
+        sheets.append(each.id)
+
+    zones = TransectZone.objects.filter(transect_id=sheets[0])
+    zones = zones.values(
+        'transect__school', 'transect__date_time', 'transect__site__site_name',
+        'transect__weather', 'transect__slope', 'transect__notes', 'zone_num',
+        'conifers', 'hardwoods', 'shrubs', 'comments'
+    )
+    sheets.pop(0)
+
+    for sheet in sheets:
+        n_zones = TransectZone.objects.filter(transect_id=sheet)
+        n_zones = n_zones.order_by('transect__id', 'zone_num').values(
+            'transect__school', 'transect__date_time',
+            'transect__site__site_name', 'transect__weather',
+            'transect__slope', 'transect__notes', 'zone_num', 'conifers',
+            'hardwoods', 'shrubs', 'comments'
+        )
+        zones = zones | n_zones
+
+    filename = 'riparian_transect_export'
+
+    return render_to_csv_response(
+        zones, filename=filename, field_header_map={
+            'transect__school': 'school', 'transect__date_time': 'date/time',
+            'transect__site__site_name': 'site',
+            'transect__weather': 'weather', 'transect__slope': 'slope',
+            'transect__notes': 'notes', 'zone_num': 'zone number'
+        }
+    )
+
+
+def export_cc(request, site_slug):
+    site = Site.objects.get(site_slug=site_slug)
+    canopyc = Canopy_Cover.objects.filter(site_id=site.id)
+
+    response = HttpResponse(content_type='text/csv')
+    response['Content-Disposition'] = 'attachment; \
+                                       filename="canopy_cover_export.csv"'
+    writer = csv.writer(response)
+    writer.writerow(['school', 'date/time', 'site', 'weather', 'north',
+                     'east', 'south', 'west', 'estimated canopy cover'])
+
+    sheets = []
+    for each in canopyc:
+        sheets.append(each.id)
+
+    for sheet in sheets:
+        cc = Canopy_Cover.objects.get(id=sheet)
+
+        # Convert int to binary
+        north = "{0:b}".format(cc.north_cc)
+        east = "{0:b}".format(cc.east_cc)
+        south = "{0:b}".format(cc.south_cc)
+        west = "{0:b}".format(cc.west_cc)
+
+        # "Count" the number of shaded squares
+        n = north.count('1')
+        e = east.count('1')
+        s = south.count('1')
+        w = west.count('1')
+
+        writer.writerow([cc.school, cc.date_time, cc.site.site_name,
+                         cc.weather, n, e, s, w, cc.est_canopy_cover])
+
+    return response
```

Comment:
From what I can tell, ``render_to_csv_response`` directly takes values from a django queryset and puts them into a csv. Since I'm modifying the integer fields to give an actual count, I have to output canopy cover csvs differently.

If you'd like a little more background to double-check my reasoning:
*  [More on djqscsv](https://github.com/azavea/django-queryset-csv)
* [Outputting to a CSV w/ Django](https://docs.djangoproject.com/en/1.10/howto/outputting-csv/)